platforms/Linux/DEB/Ubuntu/focal/debian/copyright

```diff
@@ -0,0 +1,38 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
Let's add a heading 'TODO' or something to make it clear this needs more work before releasing packages.

Reply:
@jblache is this copyright file intended to reflect the packaging scripts sources or the swift sources?

Reply:
@tomerd Both!